### PR TITLE
feat(presetter): add default header presetter

### PR DESCRIPTION
**Pull request by liubog2008**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR fixes** *(optional, close the issue(s) when PR gets merged)*:

Fixes #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by ddysher** [comment]
what's the status?

**Comment by liubog2008** [comment]
？？？

File: types/roundtrip.go [hunk 1]
```diff
@@ -42,6 +42,10 @@ type Response struct {
 	// StatusCode checks response code
 	StatusCode int `json:"statusCode"`
 
+	// Headers defines http header of request
+	// NOTE(liubog2008): whether to use map[string][]string
```
Comment (ddysher) [inline types/roundtrip.go]: do you mean we should try using `map[string][]string` here?

Comment (liubog2008) [reply types/roundtrip.go]: yes

**Comment by ddysher** [comment]
@liubog2008 @zjj2wry  you guys need to approve the PR

**Pull request merged**